### PR TITLE
chore: Improve error message when misimporting in Workflow code

### PR DESCRIPTION
**Pull request by lorensr**
To help address users getting stuck like this (despite the warning about async_hooks that appears in Worker output): https://temporalio.slack.com/archives/C01DKSMU94L/p1652371946624169

Currently not working (see image)—wondering if there's a better condition to test?

image

### CODE REVIEW

File: packages/activity/src/index.ts [hunk 1]
```diff
@@ -26,7 +26,13 @@
  * @module
  */
 
-import { msToNumber } from '@temporalio/internal-workflow-common';
+if ((globalThis as any).__TEMPORAL__ !== undefined) {
```
Comment (bergundy) [inline packages/activity/src/index.ts:29]: Wondering if it's worth making this a function in `common`, not critical

File: packages/activity/src/index.ts [hunk 2]
```diff
@@ -26,7 +26,13 @@
...
+  throw new IllegalStateError(
+    "You are importing from '@temporalio/activity' in your Workflow code, which doesn't work. Workflow code should only import from '@temporalio/workflow' and '@temporalio/common'."
```
Comment (bergundy) [inline packages/activity/src/index.ts:31]: How about something like:
```suggestion
    "Importing from '@temporalio/activity' from Workflow code is not supported. Workflow code should only import from '@temporalio/workflow' and '@temporalio/common'. More information here: https://docs.temporal.io/docs/typescript/determinism/"
```

Comment (bergundy) [reply packages/activity/src/index.ts:31]: I'm not a fan of this personal tone in error messages

**Comment by lorensr** [comment]
TODO: add test of failing import/bundling

**Comment by bergundy** [comment]
Closing this for now since using non workflow libs should result in failed bundle creation.
We should probably add the forbidden `@temporalio/*` libs to the list of blacklisted modules.

**Pull request closed without merging**